build.sh

```diff
@@ -153,15 +153,15 @@ build_pi_kernel() {
 
         make clean
 
-        yes "" | KERNEL=${KERNEL} ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} make oldconfig || exit 1
+        yes "" | KERNEL=${KERNEL} ARCH=${ARCH} CROSS_COMPILE="ccache ${CROSS_COMPILE}" make oldconfig || exit 1
 
-        KERNEL=${KERNEL} ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} make -j $J_CORES zImage modules dtbs || exit 1
+        KERNEL=${KERNEL} ARCH=${ARCH} CROSS_COMPILE="ccache ${CROSS_COMPILE}" KBUILD_BUILD_TIMESTAMP='' make -j $J_CORES zImage modules dtbs || exit 1
```

Comment:
I'm not sure about the `KBUILD_BUILD_TIMESTAMP=''`
I've got this hint from: https://nickdesaulniers.github.io/blog/2018/06/02/speeding-up-linux-kernel-builds-with-ccache/